src/numberReducer.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/**
+ * Implement `numberReducer` with 3 actions:
+ * 1. `increase`
+ * 2. `decrease`
+ * 3. `clear`
+ *
+ * If action not declared, it should return actual state.
+ * Remember that it is necessary to set initial value for the state.
+ *
+ * Example 1: `numberReducer(state, action) === newState`
+ * state === { count: 0 }
+ * action === { type: 'increase' };
+ * newState === { count: 1 };
```

Comment:
`numberReducer({ count: 0 }, { type: 'increase' })` should return `{ count: 1 }`

Reply:
the state is just a number here not an object